src/htm/regions/SPRegion.cpp

```diff
@@ -55,7 +54,7 @@ SPRegion::SPRegion(const ValueMap &values, Region *region)
   args_.minPctOverlapDutyCycles = values.getScalarT<Real32>("minPctOverlapDutyCycles", 0.001f);
   args_.dutyCyclePeriod = values.getScalarT<UInt32>("dutyCyclePeriod", 1000);
   args_.boostStrength = values.getScalarT<Real32>("boostStrength", 0.0f);
-  args_.seed = values.getScalarT<Int32>("seed", 1);
+  args_.seed = values.getScalarT<Int32>("seed", -1);
```

Comment:
why -1? Rather keep no-change, as this shouldn't affect this PR in any way.

Reply:
>> why -1? Rather keep no-change, 

Again, the default actually used was -1 from the Spec.  I changed it here to match so that the test results would not be affected now that the default is now taken from here.